Add new label to stability runs

DrewScoggins opened the pull request:
This change adds stability runs to the new Windows Server 2016 label as
well as increasing the number of iterations to 11 and dropping the
first.  We have also added the same stability prefix that we are using
for regular performance runs to try and remove as much noise as possible
from the benchmark.

DrewScoggins committed:
Add new label to stability runs

This change adds stability runs to the new Windows Server 2016 label as
well as increasing the number of iterations to 11 and dropping the
first.  We have also added the same stability prefix that we are using
for regular performance runs to try and remove as much noise as possible
from the benchmark.

dnfclas commented:
@DrewScoggins,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by .NET Foundation. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

DrewScoggins commented:
test ci please

DrewScoggins commented:
@jorive PTAL

jorive commented on stability/runPythonOnWindows.bat:
```diff
@@ -1,11 +1,11 @@
-py %WORKSPACE%\stability\stability\windows_native-stability-test.py --stabilization --iterations=10
+START "CORECLR_PERF_RUN" /B /WAIT /HIGH /AFFINITY 0x2 py %WORKSPACE%\stability\stability\windows_native-stability-test.py --iterations=11
```
I think we should have this inside the stability test wrapper, just when we launch the actual process.

DrewScoggins committed:
Update the way that we use affinity and priority

Move the call from the the invocation of the pythin script to the actual
call of the benchmark in the stability test.

jorive commented on stability/stability/windows_native-stability-test.py:
```diff
@@ -148,7 +148,7 @@ def runParsec():
     downloadAndUnpack(benchmarkLocation, targetFile)
     
-    return runAndProcess("%WORKSPACE%\\blackscholes_cpp_serial 1 in_10M.txt prices.txt", parsecProcessResults)
+    return runAndProcess("START \"STABILITY_PERF_RUN\" /B /WAIT /HIGH /AFFINITY 0x2 %WORKSPACE%\\blackscholes_cpp_serial 1 in_10M.txt prices.txt", parsecProcessResults)
```
I think you need to prefix the command with `cmd.exe /c`. START is a cmd.exe command and might now be understood outside the cmd.exe process.

DrewScoggins replied:
When I tested this locally it launches correctly.  This works because we are using shell=True so we get shell commands when we run the command line passed.

DrewScoggins commented:
Does everything else look good, @jorive?

jorive commented:
Everything else looks good. Thanks :)

DrewScoggins commented:
test ci please
